client/account-recovery/lost-password/lost-password-form/style.scss

```diff
@@ -0,0 +1,22 @@
+.lost-password-form__title {
+	font-size: 1.5em;
+	font-weight: bold;
+}
+
+//Use a more specific selector here so we don't have to use !important to overide any css
+.lost-password-form__user-login-input.form-text-input {
+	margin-top: 5px;
+}
+
+.lost-password-form__instruction-list {
+	margin: 10px 0px 10px 20px;
```

Comment:
Minor, but we do all our margins in increments of 8 generally. Same with the 5px margin above, could be 4px instead. cc @evilluendas